net.nokyan.Resources.json

```diff
@@ -0,0 +1,139 @@
+{
+    "id": "net.nokyan.Resources",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "45",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-nightly",
+        "org.freedesktop.Sdk.Extension.llvm16"
+    ],
+    "command": "resources",
+    "finish-args": [
+        "--device=dri",
+        "--env=G_MESSAGES_DEBUG=none",
+        "--env=RUST_BACKTRACE=full",
+        "--env=RUST_LOG=resources=debug",
+        "--env=XDG_DATA_DIRS=/app/share:/usr/share:/usr/share/runtime/share:/run/host/user-share:/run/host/usr/share:/run/host/share:/var/lib/flatpak/exports/share:~/.local/share/flatpak/exports/share",
+        "--filesystem=/var/lib/flatpak/app:ro",
+        "--filesystem=/var/lib/flatpak/exports/share:ro",
+        "--filesystem=xdg-data/flatpak/app:ro",
+        "--filesystem=xdg-data/flatpak/exports/share:ro",
+        "--filesystem=host",
```

Comment:
that broad and redundant fielsystem permissions at the same time.

Reply:
How are they redundant exactly? They are needed for finding flatpaks and when removing one of them, parts of app detection don't work properly. E. g. if I remove filesytem=host, only flatpak apps are recognized